components/atoms/Radio/radio-group.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import clsx from "clsx";
+
+import { BsFillCheckCircleFill } from "react-icons/bs";
+// import { MdOutlineRadioButtonUnchecked } from "react-icons/md";
+
+import * as RadioGroupPrimitive from "@radix-ui/react-radio-group";
+
+const RadioGroup = React.forwardRef<
+  React.ElementRef<typeof RadioGroupPrimitive.Root>,
+  React.ComponentPropsWithoutRef<typeof RadioGroupPrimitive.Root>
+>(({ className, ...props }, ref) => {
+  return <RadioGroupPrimitive.Root className={clsx("grid gap-2", className)} {...props} ref={ref} />;
+});
+RadioGroup.displayName = RadioGroupPrimitive.Root.displayName;
+
+const RadioGroupItem = React.forwardRef<
+  React.ElementRef<typeof RadioGroupPrimitive.Item>,
+  React.ComponentPropsWithoutRef<typeof RadioGroupPrimitive.Item>
+>(({ className, ...props }, ref) => {
+  return (
+    <RadioGroupPrimitive.Item
+      ref={ref}
+      className={clsx(
+        "aspect-square h-4 w-4 rounded-full border border-primary text-sauced-orange shadow focus:outline-none focus-visible:ring-1 focus-visible:ring-orange-200 disabled:cursor-not-allowed disabled:opacity-50",
+        className
+      )}
+      {...props}
+    >
```

Comment:
Refactor (blocking): Remove the `focus-visible` classes from the `<RadioGroupPrimitive.Item />` component.

```suggestion
    <RadioGroupPrimitive.Item
      ref={ref}
      className={clsx(
        "aspect-square h-4 w-4 rounded-full border border-primary text-sauced-orange shadow disabled:cursor-not-allowed disabled:opacity-50",
        className
      )}
      {...props}
    >
```

Add add a selector to the globals.css file for global focus styles for the radio buttons. Radix radiot buttons are buttons with `role="radio"`.

```diff
 input[type="checkbox"]:focus-visible,
- [role="checkbox"]:focus-visible {
+ [role="checkbox"]:focus-visible,
+ [role="radio"]:focus-visible {
   @apply outline-none;
   @apply ring;
   @apply border-orange-500; 
   @apply ring-orange-100;
 }
```